samply/src/linux_shared/mod.rs

```diff
@@ -1203,9 +1203,14 @@ where
             }
         }
 
-        let base_svma = match file.segments().next() {
-            Some(first_segment) => first_segment.address(),
-            None => 0,
+        let is_pe = file.format() == object::BinaryFormat::Pe;
+        let base_svma = if is_pe {
+            file.relative_address_base()
+        } else {
+            match file.segments().next() {
+                Some(first_segment) => first_segment.address(),
+                None => 0,
+            }
```

Comment:
It might be better to just call into `samply_symbols::relative_address_base` here. That way we'll know that whatever we do here is consistent with what symbolication does.